_test/test_sqw_pixels/test_PixelData.m

```diff
@@ -2,7 +2,7 @@
 
     properties
 
-        SMALL_PG_SIZE = 1e6;  % 1Mb
+        SMALL_PG_SIZE = 1e5;  % 100,000 pix % This disables filebased access and makes tested files memory based. Not what is intended in tests
```

Comment:
Is this saying that the test does not work? Guess we should rejig the test then. If it does work then maybe the comment should say something different.